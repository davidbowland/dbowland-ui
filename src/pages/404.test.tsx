import React from 'react'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import NotFound from './404'

describe('404 error page', () => {
  test('Ensure snapshot match', () => {
    const { container } = render(<NotFound />)
    expect(container.firstChild).toMatchSnapshot()
  })

  test('Ensure title is on the page', () => {
    const { getByText } = render(<NotFound />)
    expect(getByText('404: Not Found')).toBeInTheDocument()
  })
})
