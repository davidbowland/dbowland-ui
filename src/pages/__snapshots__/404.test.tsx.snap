// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 error page Ensure snapshot match 1`] = `
<main
  class="sc-dlnjwi gnIxof"
>
  <h1
    class="sc-gtsrHT elXEts"
  >
    404: Not Found
  </h1>
  <p
    class="sc-bdnxRM fSObbe"
  >
    The resource you requested is unavailable. If you feel you have reached this page in error, please contact the webmaster.
    <br />
    <br />
    <a
      href="/"
    >
      Go home
    </a>
    .
  </p>
</main>
`;
